[2024-12-19T08:51:45.886+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-19T08:51:45.919+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v2.first_task scheduled__2021-11-06T00:00:00+00:00 [queued]>
[2024-12-19T08:51:45.936+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v2.first_task scheduled__2021-11-06T00:00:00+00:00 [queued]>
[2024-12-19T08:51:45.943+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 6
[2024-12-19T08:51:46.085+0000] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): first_task> on 2021-11-06 00:00:00+00:00
[2024-12-19T08:51:46.110+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=2797) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-12-19T08:51:46.114+0000] {standard_task_runner.py:72} INFO - Started process 2809 to run task
[2024-12-19T08:51:46.119+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v2', 'first_task', 'scheduled__2021-11-06T00:00:00+00:00', '--job-id', '549', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmp4agv487x']
[2024-12-19T08:51:46.134+0000] {standard_task_runner.py:105} INFO - Job 549: Subtask first_task
[2024-12-19T08:51:46.279+0000] {task_command.py:467} INFO - Running <TaskInstance: our_first_dag_v2.first_task scheduled__2021-11-06T00:00:00+00:00 [running]> on host 8d142e9c83a7
[2024-12-19T08:51:46.489+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='mateo' AIRFLOW_CTX_DAG_ID='our_first_dag_v2' AIRFLOW_CTX_TASK_ID='first_task' AIRFLOW_CTX_EXECUTION_DATE='2021-11-06T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-11-06T00:00:00+00:00'
[2024-12-19T08:51:46.491+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-19T08:51:46.540+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2024-12-19T08:51:46.544+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hello world, this is the first task']
[2024-12-19T08:51:46.591+0000] {subprocess.py:99} INFO - Output:
[2024-12-19T08:51:46.597+0000] {subprocess.py:106} INFO - hello world, this is the first task
[2024-12-19T08:51:46.599+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2024-12-19T08:51:46.672+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-19T08:51:46.675+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v2, task_id=first_task, run_id=scheduled__2021-11-06T00:00:00+00:00, execution_date=20211106T000000, start_date=20241219T085145, end_date=20241219T085146
[2024-12-19T08:51:46.949+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-19T08:51:46.994+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-19T08:51:47.006+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
